src/routes/adminSettings/index.js

```diff
@@ -27,6 +27,7 @@ export default function ({ db, ensurePfp }) {
     if (!req.user.admin) return res.redirect('/')
 
     const username = req.body.newUserUsername.trim()
+    const ssoUser = req.body.isSSOUser
```

Comment:
What does ssoUser do here? Also we should be storing booleans rather than strings. Please make sure not to lock in Google as the only SSO IDP. I'd also be interested in adding Discord login.

Reply:
By default, when you add a user we create an account confirmation link. This link is used by the new user to "accept" their new account and get them to set a password.

There is a change in the Admin Settings UI which adds a checkbox to the add user form. This checkbox sets this value. It is then used [here](https://github.com/cj13579/Christmas-Community/blob/google-oicd/src/routes/adminSettings/index.js#L48) to change the behavior of how we save a user. When "on", we create a user _without_ the signup token or expiry.

Reply:
I've also updated this to be a boolean: https://github.com/Wingysam/Christmas-Community/pull/138/commits/23e4a8c3310fce6ded71b53e998ea8578c3970a4.

Reply:
Okay thanks!